packages/backend/src/graphql/resolvers/outgoing_payment.ts

```diff
@@ -248,6 +250,20 @@ export function paymentToGraphql(
     createdAt: new Date(+payment.createdAt).toISOString(),
     quote: quoteToGraphql(payment.quote),
     grantId: payment.grantId,
-    tenantId: payment.tenantId
+    tenantId: payment.tenantId,
+    cardDetails: cardDetailsToGraphql(payment.cardDetails)
+  }
+}
+function cardDetailsToGraphql(
+  cardDetails?: OutgoingPaymentCardDetails
```

Comment:
Instead of having this optional parameter let's just optionally call this in `paymentToGraphql`

https://github.com/interledger/rafiki/blob/c41a4414af89aac2b326fc6442d73d5df938d815/packages/backend/src/graphql/resolvers/accounting_transfer.ts#L47-L49